source/expectations/ut_expectation_compound.tpb

```diff
@@ -76,6 +75,36 @@ create or replace type body ut_expectation_compound as
     return l_result;
   end;
 
+  member function to_include(a_expected sys_refcursor) return ut_expectation_compound is
+    l_result ut_expectation_compound := self;
+  begin
+    l_result.matcher := ut_include(a_expected);
+    return l_result;
+  end;
+
+  member function to_contain(a_expected sys_refcursor) return ut_expectation_compound is
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this weak REF CURSOR by one with a return type. ([plsql:WeakRefCursorCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3AWeakRefCursorCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=774&issues=AWbTzT_kO08i52vd_XcS&open=AWbTzT_kO08i52vd_XcS)

[ISSUE_KEY]: AWbTzT_kO08i52vd_XcS